xml/System.Net.NetworkInformation/PhysicalAddress.xml

```diff
@@ -464,11 +490,29 @@
         </Parameter>
       </Parameters>
       <Docs>
-        <param name="address">To be added.</param>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="address">A span containing the address that will be used to initialize the <see cref="T:System.Net.NetworkInformation.PhysicalAddress" /> instance returned by this method.</param>
+        <param name="value">When this method returns, contains the <see cref="T:System.Net.NetworkInformation.PhysicalAddress" /> instance equivalent of the address contained in <paramref name="address" />, if the conversion succeeded, or <see langword="null" /> if the conversion failed. This parameter is passed uninitialized; any value originally supplied in <paramref name="result" /> will be overwritten.</param>
+        <summary>Converts the span representation of a hardware address to <see cref="T:System.Net.NetworkInformation.PhysicalAddress" />. A return value indicates whether the conversion succeeded.</summary>
```

Comment:
```suggestion
        <summary>Tries to convert the span representation of a hardware address to a <see cref="T:System.Net.NetworkInformation.PhysicalAddress" /> instance. A return value indicates whether the conversion succeeded.</summary>
```

Reply:
Unlike TryParse, Parse will throw on failure right @aik-jahoda ?

Reply:
Yes, it is documented there: https://github.com/dotnet/dotnet-api-docs/blob/144d048a1a4efde7527f559d30af3a189bc77564/xml/System.Net.NetworkInformation/PhysicalAddress.xml#L344